app/server/models/credit_transfer.py

```diff
@@ -256,16 +270,22 @@ def resolve_as_complete(self, batch_uuid=None):
 
     def resolve_as_rejected(self, message=None):
         if self.transfer_status not in [None, TransferStatusEnum.PENDING]:
-            raise Exception(f'Transfer resolve function called multiple times for transaciton {self.id}')
+            raise Exception(f'Resolve called multiple times for transfer {self.id}')
 
         if self.fiat_ramp and self.transfer_type in [TransferTypeEnum.DEPOSIT, TransferTypeEnum.WITHDRAWAL]:
             self.fiat_ramp.resolve_as_rejected()
 
         self.resolved_date = datetime.datetime.utcnow()
         self.transfer_status = TransferStatusEnum.REJECTED
+        self.update_balances()
```

Comment:
update balances after resolution, always

Reply:
can you explain the reasoning as to why we would want to do this?

Reply:
i.e. why weren't we doing this before?

Reply:
oh because previously pending transactions didn't change the balance, so there was no need to update after